app/src/pages/_app.tsx

```diff
@@ -48,6 +61,13 @@ function App({ Component, pageProps }: AppProps) {
               <Navbar.Section grow pt={18}>
                 <ResourceCounts />
               </Navbar.Section>
+              <Navbar.Section px={18} style={{ paddingBottom: 16 }}>
+                <Link href={`/authoring`} style={{ width: '100%' }}>
```

Comment:
Is there anything we can do to make this authoring button light up while we are on an authoring page? Perhaps make it that primary blue color or maybe the 'warning' color if that exists in mantine. It would help to know you are in authoring mode.